test/benchmark_http_client_test.cc

```diff
@@ -261,6 +275,7 @@ TEST_F(BenchmarkClientHttpTest, BadContentLength) {
   };
 
   // Note we we explicitly do not expect encodeData to be called.
+  EXPECT_CALL(stream_encoder_, encodeData(_, _)).Times(0);
```

Comment:
(Optional) now that we actually have an explicit expectation in place instead of an implied one, we could probably do without the comment above this line?

Reply:
Done.

Thank you Otto for the review. 
Verified the change pass all unit tests except termination_predicate_test which is failing at master branch.

bazel test test:all

Reply:
Great, thank you, this looks good to me. As for the termination_predicate_test failure, could you paste the details here or in a separate issue? Maybe it is flaking (I can't find an occurrence of that, neither on master).

Reply:
Thanks Otto. 
Opened an issue here
https://github.com/envoyproxy/nighthawk/issues/372